pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -6637,7 +6637,12 @@ def solve_model(model, options=None):
             # In the post-process solve, only the hydraulics block is solved.
 
             mh = model_h.hydraulics
-            results_2 = opt.solve(mh, tee=True)
+            # Calculate hydraulics. The following conditional is used to avoid errors when
+            # using gurobi vs gurobi_direct solvers)
```

Comment:
```suggestion
            # Calculate hydraulics. The following condition is used to avoid attribute error when
            # using gurobi_direct on hydraulics sub-block
```